Mathlib/Data/Fintype/Pi.lean

```diff
@@ -174,3 +175,94 @@ lemma piDiag_subset_piFinset [DecidableEq ι] [Fintype ι] :
     s.piDiag ι ⊆ Fintype.piFinset fun _ ↦ s := by simp [← piFinset_filter_const]
 
 end Finset
+
```

Comment:
Could you add a section heading (`/-! ### title -/`) here? Perhaps "Results about finsets of tuples" or similar?